openedx/frontend-app-ora-grading
# feat: open response assesment detail (problem steps)  UI

### Description
This PR allows us to see in detail each step of a open response, which means 'Training,' 'Peer,' 'Self,' 'Staff,' and can also see feedback received and feedback given

### Further information
https://github.com/openedx/edx-ora2/issues/1974


![ora-frontend-dem](https://github.com/openedx/frontend-app-ora-grading/assets/134975835/be5622d7-126e-44ad-8aad-223fbcfa78d9)

BEFORE           |  NOW
:-------------------------:|:-------------------------:
![](https://github.com/openedx/frontend-app-ora-grading/assets/134975835/ddb41876-d95f-44fe-9e65-6dc6a13f7a2a)  |  ![](https://github.com/openedx/frontend-app-ora-grading/assets/134975835/08e2fee6-3b99-4eed-8033-cb74fe8641e0)


#### Responses list
![responses](https://github.com/openedx/frontend-app-ora-grading/assets/134975835/c67b8099-dae9-42ee-99d1-a9c225236d22)


#### Assesment list
![now-ora-fix](https://github.com/openedx/frontend-app-ora-grading/assets/134975835/08e2fee6-3b99-4eed-8033-cb74fe8641e0)

#### Assesment feedback received
![assesment-received](https://github.com/openedx/frontend-app-ora-grading/assets/134975835/f6c6901e-886f-413c-94c2-1e304d59ad4c)


#### Assesment feedback given

![assesment-given](https://github.com/openedx/frontend-app-ora-grading/assets/134975835/fe8ae44d-aede-41f1-a882-f611f6e5d0d8)


Here's what you see in the video explained:

- There is an ORA open response type "Staff Assessment Only."
- The instructor opens the new experience of ORA that contains the submissions of that Open response.
- The instructor can see the details and more information, such as email, full name, list of responses, and feedback in detail.

 
####  Why are we proposing these changes?

This improvement enables instructors to access detailed information for each Open Response, making it easier for them to review ORA responses and maintain better control over the feedback process.

### How to test

- Create an unit with an open response type **"Staff Assessment Only."**
- Submit the response
- Go to **instructor** tab
- In the unit click in  **"View and grade responses"**

Now you can see the UI implemented


**NOTE:** it depends on this PR: https://github.com/openedx/edx-platform/pull/33632
